spid_compliant_certificates/validator/validate.py

```diff
@@ -70,43 +43,42 @@ def validate(crt_file: str, sector: str) -> None:
     else:
         raise Exception(msg)
 
+    r = Report(str(crt_file.absolute()))
+
     # check key type and size
-    results.append(_do_check(
+    r.add_test(_do_check(
         checks.key_type_and_size(crt),
         'Checking the key type and size'
     ))
 
     # check digest algorithm
-    results.append(_do_check(
+    r.add_test(_do_check(
```

Comment:
r collides with pdb return command
https://docs.python.org/3/library/pdb.html#pdbcommand-return

better use something different

Reply:
Really? Are you saying `pdb` can't be used if your code defines a variable named `r`? Anyway, I'll change to `rep.add_test()`.

Reply:
... then it would need some trick like `!r` but better don't collide :) thank you

Reply:
If everything is ok, could you please resolve the conversation?